docs/docs/user/migration-guide/v6-v7.md

```diff
@@ -60,3 +60,42 @@ As a result of the restructure, some imports need to be adjusted:
 -components/__init__.py
 +kpops/components/custom/__init__.py
 ```
+
+## [Call destroy from inside of reset or clean](https://github.com/bakdata/kpops/pull/501)
+
+Before v7, the KPOps CLI executed `destroy` before running `reset/clean` to ensure the component was destroyed.
+
+This logic has changed. The `destroy` method is now called within the `PipelineComponent`'s `reset/clean`.
+
+During migrating to v7, you should check your custom components and see if they override the `reset/clean` methods. If so, you need to call the supermethod `reset/clean` to trigger the `destroy` inside the parent class. Alternatively, if you are implementing the `PipelineComponent` class, you need to call the `destroy` method at the beginning of the method.
+
+#### components.py
+
+For example, when creating a custom `StreamsApp` or `ProducerApp`, you **must** call the supermethod `rest/clean` to execute the `destroy` in the parent class. **Otherwise, the logic of destroy will not be executed!**
+
+```diff
+class MyStreamsApp(StreamsApp):
+    
+    @override
+    async def clean(self, dry_run: bool) -> None:
++       await super().clean(dry_run)
+        # Some custom clean logic
+        # ...
+```
+
+If you implement the `PipelineComponent` you **must** call the `destroy` method inside the `clean` method.
```

Comment:
now we can change this paragraph, right?

Reply:
I have changed it :)